app/controllers/organizations/organization_profiles_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Organizations::OrganizationProfilesController < ApplicationController
+  before_action :verified_staff
```

Comment:
```suggestion
  layout "dashboard"
  before_action :require_organization_admin
```
This should secure actions to org admins only and polish the views! :)